sql/src/test/java/io/crate/analyze/CreateAlterTableStatementAnalyzerTest.java

```diff
@@ -1030,6 +1030,62 @@ public void testCreateTableGeneratedColumnBasedOnUnknownColumn() {
             "   date_string as cast(unknown_col as string))");
     }
 
+    @Test
+    @SuppressWarnings("unchecked")
+    public void testCreateTableWithDefaultExpressionLiteral() {
+        CreateTableAnalyzedStatement analysis = e.analyze(
+            "create table foo (name text default 'bar')");
+
+        Map<String, Object> mappingProperties = analysis.mappingProperties();
```

Comment:
shall we use `mapToSortedString` like in other tests? it probably would make tests a bit more readable.